hardware_interface/src/component_parser.cpp

```diff
@@ -45,6 +45,7 @@ constexpr const auto kTypeAttribute = "type";
 constexpr const auto kRoleAttribute = "role";
 constexpr const auto kReductionAttribute = "mechanical_reduction";
 constexpr const auto kOffsetAttribute = "offset";
+constexpr const auto kInitialValueAttribute = "initial_value";
```

Comment:
And please move in the line 41 so that is near min/max definitions.

```suggestion
constexpr const auto kInitialValueTag = "initial_value";
```

Reply:
Done.